src/runner.js

```diff
@@ -117,12 +117,14 @@ export class Runner {
     // create context
     if (this.options.incognito) {
       this.browser = await pw.launch(pwOptions)
-      this.context = await this.browser.newContext()
-    } else {
-      this.context = await pw.launchPersistentContext(
-        this.browserDir,
-        pwOptions
+      this.context = await this.browser.newContext(
+        this.options.browserContextOptions
       )
+    } else {
+      this.context = await pw.launchPersistentContext(this.browserDir, {
+        ...pwOptions,
+        ...this.options.browserContextOptions,
```

Comment:
probably need to add a default value like `{}` for this setting

Reply:
Do I? https://runkit.com/marcopolo/obj-spread

Reply:
i would prefer to have it there, for consistency